dit/divergences/tests/test_generalized_divergences.py

```diff
@@ -96,11 +96,11 @@ def test_divergences_to_kl():
         for dist1 in dists:
             for dist2 in dists:
                 assert_almost_equal(alpha_divergence(dist1, dist2, alpha=-1), kullback_leibler_divergence(dist2, dist1))
-                if dist1 != dist2:
+                if dist1 is not dist2:
```

Comment:
we do not care about order.

i'll update it to use combinations later. i think there's a spot in MUIProfile that could benefit from this too.